README.md

```diff
@@ -1,2 +1,33 @@
 # geo-arrow-spec
-Specifications for storing geospatial data in Apache Arrow and Apache Parquet
+
+Specifications for storing geospatial data in Apache Arrow and Apache Parquet.
+
+The [Apache Arrow](https://arrow.apache.org/) project specifies a standardized
+language-independent columnar memory format. It enables shared computational
+libraries, zero-copy shared memory and streaming messaging, interprocess
+communication, etc and is supported by many programming languages.
+
+[Apache Parquet](https://parquet.apache.org/) is an efficient, columnar storage
+format (originating from the Hadoop ecosystem). It is a widely used file format
+for tabular data, and its implementation for C-based languages (Python, R) is
+included in the Apache Arrow project.
+
+Geospatial data often comes in tabular format, with one (or multiple) column
+with feature geometries and additional columns with feature attributes. 
+
+Defining a standard way to store geospatial data in the Arrow memory layout
+can help interoperability between different tools and enable fast data
+transfer:
+
+- Ensure that different tools can read Parquet or Feather files with geospatial
+  data and correctly interpret the geometry information. Concrete example: we
+  want that the R `sf` package can read a parquet file produced by the Python
+  `geopandas` package and vice versa.
```

Comment:
A simple description of Feather might be helpful.

Reply:
Added a short sentence about it in the paragrapgh about Apache Arrow.